CONTRIBUTING.md

```diff
@@ -35,19 +35,19 @@ enum class FetchOption {
 
 ### File Headers
 
-Every `.cpp`, `.h` and `.inl` file must contain the following header at the very top:
+Make sure every `.cpp`, `.h` and `.inl` file contains the following header at the very top:
 
 ```cpp
 /*
```

Comment:
This licence headers comes from our OSS office and was provided by the legal department, means we cannot modify it. Please change back to original version (even if it contains typos :( )

Reply:
Noted, thanks